internal/cli/cmd/connectivity.go

```diff
@@ -66,11 +66,12 @@ func newCmdConnectivityTest() *cobra.Command {
 
 			// Instantiate the test harness.
 			cc, err := check.NewConnectivityTest(k8sClient, params)
+
 			if params.PerfSamples > 3 {
-				cc.Warn("Having Performance Samples > 3 can cause issues")
+				cc.Warn("Having Performance samples > 3 can cause issues")
```

Comment:
Is it worth documenting what the "issues" are? Such documentation would be helpful for future users of the code.

Also, for CRR, `params.PerfSamples` and `params.PerfDuration` are ignored, so the warning could be misleading here, as the values supplied by the user are not actually used.

Reply:
ah, happy to. I linked the issue that this PR fixes : https://github.com/cilium/cilium-cli/issues/749 

Samples / Duration are only not applied if the user passes `-perf-crr. `

Reply:
ah! I had to look again, i see your point!

Reply:
With CRR being an opinionated workload (not allowing the user to modify the samples,duration) we can effectively remove these warnings.